cheatsheets/Java_Security_Cheat_Sheet.md

```diff
@@ -756,7 +768,10 @@ class App {
 
         // Alice encrypts a message to send to Bob
         String plaintext = "Hello, Bob!";
-        String metadata = "Info about the message";
+        
+        // Add some relevant context about the encrypted data that should be verified
+        // on decryption
+        String metadata = "Sender: alicesmith";

```

Comment:
As mentioned earlier, this often is either a DN or an email. To illustrate the point, using something like 'alice@example.com' rather than 'alicesmith' would be preferred here. Email addresses are unique; combination of a givenname / surname, not so much.

Reply:
Made into an email